msteams-platform/apps-in-teams-meetings/teams-live-share-faq.md

```diff
@@ -38,7 +38,7 @@ For more information, see [how to guide](./teams-live-share-how-to/how-to-custom
 
 <summary><b>How long is data stored in Live Share's hosted service accessible?</b></summary>
 
-Any data sent or stored through Fluid containers created by Live Share's hosted Azure Fluid Relay service might be accessible for up to 24 hours, though in most cases it's deleted within six hours. If you want to persist data beyond 24 hours, you can replace our hosted Azure Fluid Relay service with your own. Alternatively, you can use your own storage provider in parallel to Live Share's hosted service.
+Any data sent or stored through Fluid containers created by Live Share's hosted Azure Fluid Relay service might be accessible for up to 24 hours, though in most cases it's deleted within four hours. If you want to persist data beyond 24 hours, you can replace our hosted Azure Fluid Relay service with your own. Alternatively, you can use your own storage provider in parallel to Live Share's hosted service.
```

Comment:
It's still 6 hours before container deletion orchestration kicks off. So deleted within 6 hours, right?